debian/Dockerfile.tpl

```diff
@@ -17,3 +17,5 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 COPY 01_nodoc /etc/dpkg/dpkg.cfg.d/
 COPY 01_buildconfig /etc/apt/apt.conf.d/
+
+RUN mkdir -p /usr/share/man/man1
```

Comment:
@petrosagg it is excluded in this configuration file https://github.com/resin-io-library/base-images/blob/master/debian/01_nodoc#L4